pkgs/top-level/all-packages.nix

```diff
@@ -13379,6 +13379,8 @@ with pkgs;
 
   spicy = callPackage ../development/tools/spicy { };
 
+  spip = callPackage ../by-name/sp/spip { };
+
```

Comment:
This only works if there is a `default.nix` in the directory. You can omit it since it's done automatically for pkgs/by-name.

Please excuse the unreadable CI error output...

Reply:
Thanks ! By the way, I did not know all new packages should be in pkgs/by-name. Is that a new policy ? Are there moved later on ?

The few other packages I've committed were not in pkg/by-name.

Reply:
> Is that a new policy ? Are there moved later on ?

It is mentioned in the "Quick Start to Adding a Package": https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#quick-start-to-adding-a-package

Eventually there will be an automated migration (#211832).

Reply:
Oh and I meant that you could omit the change to all-packages.nix. Packages in pkgs/by-name are automatically picked up.

Reply:
Thanks for the heads-up. I've updated all-packages.